assets/app/styles/_core.less

```diff
@@ -203,6 +203,19 @@ html, body {
   }
 }
 
+.nav-tabs {
+  font-size: @font-size-base;
+  a {
+    // ui.bootstrap.tabs doesn't set the href attr, so the cursor style is
```

Comment:
Yeah, you can specify active as an attribute or in code. You can also evaluate an expression when a tab is selected using the `select` attribute.

What do you think about hash URIs for selected tabs? For instance,

`https://localhost:9000/project/project1/browse/pods/django-example-2-sq7ju#metrics`